src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.TrackingSession.cs

```diff
@@ -217,7 +217,7 @@ public void Cancel()
                     return TriggerIdentifierKind.NotRenamable;
                 }
 
-                if (!sourceSymbol.Locations.All(loc => loc.IsInSource))
+                if (!sourceSymbol.Locations.Any() || !sourceSymbol.Locations.All(loc => loc.IsInSource))
```

Comment:
Should we also jus refactor a proper "IsFromSource" helper because I think we have this pattern in other places too.